src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -591,9 +591,7 @@ public static PSDataCollection<PSObject> GetPSDataCollection(object inputValue)
         /// <param name="second">Object to compare first to.</param>
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
SemanticVersion api tests.Error handling.Missing parts: 'NullString': '$null'
```powershell
Expected an exception, with FullyQualifiedErrorId 'FormatException' to be thrown, but the FullyQualifiedErrorId was 'PSArgumentNullException'. from D:\a\1\s\test\powershell\engine\Basic\SemanticVersion.Tests.ps1:209 char:19
    +                 { [SemanticVersion]::Parse($version) } | Should -Thro ...
    +                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\SemanticVersion.Tests.ps1: line 209
209:                 { [SemanticVersion]::Parse($version) } | Should -Throw -ErrorId "FormatException"
```